README.md

```diff
@@ -15,19 +15,37 @@
 ### Tech Stack <a name="tech-stack"></a>
 
 > Simple html5, css3 with small basic js DOM project with linter configuration.
-
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/default.asp">JS</a></li>
+  </ul>
+</details>
 
 ## 💻 Key Features <a name="getting-started"></a>
 
 > The key features are the responsive
 
 - **Web view**
 - **Mobile view**
+- **[Home Page]**
+- **[About Page]**
+- **[Hero Section]**
+- **[Main Program Section]**
+- **[Speakers Section]**
+- **[Partners Section]**
+- **[Footer Section]**
+- **[Mobile menu]**
+- **[Style linters]**
+- **[Javascript Linters]**
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 > Click on the link below to see the live demo
 
-- [Live Demo Link]
+- [Live Demo Link](https://rudy-menekam.github.io/developers-community/)
```

Comment:
⚠️ [optional] 
- Kindly remember to update your live demo link after merging and include the update on your readme. 🌸

Reply:
Oh thank you i did update, but looks like it has not been deployed